maven-artifact/src/main/java/org/apache/maven/artifact/handler/ArtifactHandler.java

```diff
@@ -32,7 +32,8 @@ public interface ArtifactHandler {
     String ROLE = ArtifactHandler.class.getName();
 
     /**
-     * Get the file extension associated to the file represented by the dependency type.
+     * Returns the file name extension of the artifact;
```

Comment:
`the file name extension` -> `the default file name extension`

Reply:
no, not that one :)